catalog/dags/data_refresh/create_and_populate_filtered_index.py

```diff
@@ -0,0 +1,116 @@
+"""
+# Create and Promote Index
+
+This file contains TaskGroups related to creating and populating the filtered Elasticsearch indices
+as part of the Data Refresh.
+
+TODO: We'll swap out the create and populate filtered index DAG to use this instead
+of the one defined in the legacy_data_refresh.
```

Comment:
Noting that there is a separate issue for this, (and moreover the filtered index is going away).